README.md

```diff
@@ -133,3 +133,51 @@ Usage: ./search-circle-builds [options]
         --branch BranchName          Limit results to builds for one branch
     -h, --help                       Show this message
 ```
+
+### compute-timing-stats
+Figure out which Circle steps take the most time, in successful builds.
+
+This can give you a list of steps by average duration in descending order:
+```
+╰○ ./compute-timing-stats --branch staging --start 31000                                  
+Downloading 31000..32037 |Time: 00:00:05 | === | Time: 00:00:05
+Limiting to branch staging
```

Comment:
maybe "Limited to branch: staging" just to make it easier to parse the branch name?